cmd/ci-studio-common-util/commands/allow.go

```diff
@@ -0,0 +1,16 @@
+package commands
```

Comment:
Do we need the copyright headers?

Reply:
I've never completely understood that. We've got an uneven mix of those in a2, and I'd prefer we didn't do it if we don't have to.

Reply:
Also, if we (legally, I assume) **need** these headers, let's add something to CI to ensure they're present.

Reply:
I have no idea if we need or should have them, just a curious question

Reply:
I believe the Habitat team got an answer about this and the answer was we don't need them?  (cc @christophermaier)

Reply:
Roger that. Thank you both 💯:+1:

Reply:
No, we do not need copyright headers. The fact that we have a license file at the repository root (or can point to one) is sufficient.